README.md

```diff
@@ -9,7 +9,7 @@
 
 </div>
 
-This is a PHP library that allows interaction with [Conekta's API](https://api.conekta.io).
+This is a PHP library that allows interaction with [Conekta's API](https://developers.conekta.com/reference/autenticaci%C3%B3n).
```

Comment:
i think that `https://api.conekta.io` is the value correct, checks title 
`This is a PHP library that allows interaction with `